recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
It's usually installed from PyPI with pip ;-), but from the source on Github, the canonical way is with [flit](http://flit.readthedocs.org/en/latest/). I plan to work on packaging flit for conda, but for the time being, copying the .py file into site-packages is a perfectly adequate mechanism.